index.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+class Database {
+    private $conn;
+
+    public function __construct() {
+        $this->conn = new mysqli("localhost", "root", "password", "my_database");
+
+        if ($this->conn->connect_error) {
+            die("Connection failed: " . $this->conn->connect_error);
+        }
+    }
+
+    public function saveData($name, $description, $price) {
```

Comment:
Add type declarations to function parameters and return types:
public function saveData(string $name, string $description, float $price): void {